manifests/init.pp

```diff
@@ -28,7 +28,7 @@
 #   Defaults to 'VirtualBox' for RedHat and 'virtualbox' for Debian
 #
 class virtualbox (
-  String $version               = '5.1',
+  String $version               = '6.0',
```

Comment:
This makes this a breaking change?

Reply:
I did not tested upgrade but i am expecting that it is a breaking change because Oracle manage version of packages in a different way.

Reply:
I added more acceptance tests. And now with last push the module will upgrade from 5.x to 6.0 smoothly.

Reply:
But because the default value is changed it is a breaking change